app/layout.tsx

```diff
@@ -10,14 +10,21 @@ import { ComponentsMap } from "@/storyblok/components/ComponentsMap";
 
 // Force next.js not to cache API calls by default. This means caching is OPT-IN rather than OPT-OUT.
 // This avoids issues with the Storyblok js client, which has it's own built-in caching, and Next.js caching interferes with this...
-export const revalidate = 0;
+export const revalidate =
```

Comment:
Suggest updating the comment above to make clear we now only force non-caching in dev mode